mkdocs.yml

```diff
@@ -163,72 +163,11 @@ markdown_extensions:
 # Plugins
 plugins:
   - search
+  - awesome-pages
 
 # Page Tree/Sidebar
 # https://www.mkdocs.org/user-guide/writing-your-docs/#configure-pages-and-navigation
+# https://3os.org/utilities/markdown-cheatsheet/awesome-pages/
+# We now use awesome-pages plugin, so check `.pages` files inside renovate docs folder
 nav:
-  - Home: 'index.md'
```

Comment:
There's also a problem with the page ordering:

1. Open homepage of the docs in the preview
1. Use the <kbd>n</kbd> key on your keyboard, or select the Next button on the bottom right of the page
1. End up at the **Troubleshooting Renovate** page, instead of **Getting Started, Use Cases**